setup.cfg

```diff
@@ -1,7 +1,14 @@
+# NOTE: this package is now linted by ruff in GitHub Actions and scons;
+# this section is retained here for compatibility with users who still use
+# flake8 in their local workflows, but may not produce exactly the same
+# results as the ruff config.
```

Comment:
This phrasing is a bit awkward.  It is written as though the *users* may not produce the same results.  Something like: "This section is retained here for best-effort compatibility with flake8 for users who do not use ruff in their local workflows.  Note, however, that this may not produce exactly the same results as the ruff config which is required to pass before a PR can be accepted."

Reply:
Thanks, yeah, that's better. I'll make the same change to cfht, decam, and lsst (the latter of which was done on a different ticket).